command/command.go

```diff
@@ -122,7 +122,7 @@ func RunCommand(
 		if err != nil {
 			return err
 		}
-		fmt.Println(string(out))
+		fmt.Print(string(out))
```

Comment:
That's a good idea, I didn't think of that.

Reply:
So, how about this PR only changes this line 😅

Reply:
That would be fine with me!

Reply:
OK. it's done.